src/components/NavBar.js

```diff
@@ -2,8 +2,15 @@ import React from "react";
 
 function NavBar() {
   const links = ["home", "about", "projects"];
+  const linkElements = links.map((url) => {
+    return (
+      <a key={"#" + url} href={"#" + url}>
```

Comment:
No need for the `#` here. 
```suggestion
      <a key={url} href={"#" + url}>
```